modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,326 @@
+* Removed the Metadata configuration section. MongoDB 6.0 automatically generates the metadata.
+
+==== Config Advanced Tab
+
+The following optional changes apply to the Advanced tab in the global configuration:
+
+* Added the following read concern options:
+** Read concern: Enables the control of the consistency and isolation properties of the data read from replica sets and replica set shards.
+** Read preference: Specifies the read preferences for this connection.
+** Maximum staleness seconds: Specifies how stale a secondary can be before the client stops using it for read operations. A value of -1 means that there is no maximum staleness threshold.
+** Read preference tags: Enables you to specify a tag set in the read preference to target those members associated with a tag in the set.
+* Added the following write concern options, with the following fields:
+** Write concern acknowledgment: Requests an  acknowledgment that the write operation was propagated to the specified number of instances.
+** Write concern timeout: Specifies a time limit for the write concern.
+** Write concern timeout timeunit: Specifies a time unit for the Write concern timeout field.
+
+==== Connection Section
+
+The following changes apply to the tabs in the Connection section of the global configuration:
+
+* Added the following fields to the General tab:
+** Required Libraries: Reference to the MongoDB Driver.The version must be 3.11 or higher.
+** Server Addresses: This field is now is a list of server addresses to use for the connection.
+* Added a Security tab for optional TLS context configuration.
+* Added the following optional fields to the Advanced tab:
+** Authentication mechanism: Authentication mechanism used for this connection
+** Replica set name: Name of the replica set to which to connect
+** Authentication source: Database name associated with the user’s credentials
+** Compressors: List of compressors for which to enable network compression for communication between this client and a mongod/mongos instance
+** Zlib compression level: Integer that specifies the compression level if using zlib for network compression. Set this field to 0 for no compression or to 1-9 for compression, with 1 being the lowest compression level and 9 the highest. High compression levels require more processing time than lower compression levels.
+** Connection timeout: Connection timeout used when establishing the socket connections
+** Connection timeout time unit: Time unit for the Connection timeout field
+** Local threshold: Size of the latency window for selecting among multiple, suitable MongoDB instances
+** Local threshold time unit: Time unit for the Local threshold time unit field
+** Server selection timeout: How long a time to block for server selection before throwing an exception
```

Comment:
```suggestion
** Server selection timeout: How long to block for server selection before throwing an exception
```

Reply:
Consider whether it would be OK to omit the second-level entries, which appear to be named intuitively because these long lists make for difficult reading.